File: app/src/main/java/co/tpcreative/supersafe/common/api/response/BaseResponse.java
package co.tpcreative.supersafe.common.api.response;
import com.anjlab.android.iab.v3.PurchaseData;
import com.google.gson.Gson;
import java.io.Serializable;
import co.tpcreative.supersafe.model.Version;

public class BaseResponse implements Serializable {
    public String message;
    public String responseMessage;
    public boolean error;
    public PurchaseData purchase;
    public Version version;
    public String toFormResponse() {
        return new Gson().toJson(this);
    }
}

File: app/src/main/java/co/tpcreative/supersafe/common/api/response/BaseResponse.kt
package co.tpcreative.supersafe.common.api.response
import co.tpcreative.supersafe.model.Version
import com.anjlab.android.iab.v3.PurchaseData
import com.google.gson.Gson
import java.io.Serializable

open class BaseResponse : Serializable {
    var responseMessage: String? = null
    var responseCode : Int? = 0
    var error = false
    var purchase: PurchaseData? = null
    var version: Version? = null
}